KlingonAssistant/data/mem-16-t.xml

```diff
@@ -1570,8 +1570,8 @@ This dialect is sort of half-way between the standard dialect and the {Qotmagh S
       <column name="_id">32205</column>
       <column name="entry_name">tellar wa'</column>
       <column name="part_of_speech">n:place,fic</column>
-      <column name="definition">Tellar Prime</column>
-      <column name="definition_de"></column>
+      <column name="definition">Tellar Prime (planet name)</column>
```

Comment:
Okay, this should be consistent of course. I was just thinking that no everyone knows what that is. We also know of Optimus Prime and Spock Prime, who are not planets :-)